scene/gui/rich_text_label.cpp

```diff
@@ -2565,6 +2565,21 @@ void RichTextLabel::selection_copy() {
 	}
 }
 
+void RichTextLabel::select_all() {
+	if (!selection.enabled) {
+		return;
+	}
+
+	if (main->lines.size() > 0){
```

Comment:
Style issue: https://travis-ci.org/github/godotengine/godot/jobs/704342807

See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Thanks for the help! I just ran clang to fix that issue, rebased and force pushed. Hope now it's ok.